sig-scheduling/annual-report-2021.md

```diff
@@ -87,17 +91,18 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
 Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
 -->
 
-New in $YYYY:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
+New in 2021:
+- [kube scheduler simulator](https://github.com/kubernetes-sigs/kube-scheduler-simulator)
 -
 
-Retired in $YYYY:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
+Retired in 2021:
+- None
 -
 
 Continuing:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
--
+- [Descheduler](https://github.com/kubernetes-sigs/descheduler)
+- [Scheduler Plugins](https://github.com/kubernetes-sigs/scheduler-plugins)
+- [Cluster Capacity](https://github.com/kubernetes-sigs/cluster-capacity)
```

Comment:
@damemi @ingvagabund @ravisantoshgudimetla what is the status of the cluster capacity project? is it up to date? do we want to continue to host it?

Reply:
sg, keeping it then.